src/app/dashboard-widgets/applications-widget/applications-widget.component.html

```diff
@@ -17,30 +24,40 @@ <h3>This space has no applications</h3>
           </p>
           <div class="f8-blank-slate-main-action">
             <div *ngIf="userOwnsSpace">
-              <button id="spacehome-applications-add-button" class="btn btn-primary btn-lg" (click)="addToSpace.emit()">Create an Application</button>
+              <button id="test-applications-add-button" class="btn btn-primary btn-lg" (click)="addToSpace.emit()">Create an Application</button>
```

Comment:
Why was this ID changed and not the one on line 34?

Reply:
I just made the IDs consistent for the unit tests (i.e., all prefixed with "test"), so we know where/how they are being used.

Reply:
When you change the IDs you need to check with @ldimaggi because they might be used in the e2e. So don't remove it.

Reply:
Ok, I'll just revert the ID.